src/main/java/com/programmers/calculator/model/Operator.java

```diff
@@ -0,0 +1,33 @@
+package com.programmers.calculator.model;
+
+public enum Operator {
+    PLUS("+"){
+        @Override
+        public int calculate(int operand1, int operand2) {
+            return operand1 + operand2;
+        }
+    }, MINUS("-"){
+        @Override
+        public int calculate(int operand1, int operand2) {
+            return operand1 - operand2;
+        }
+    }, MULTIPLY("*"){
+        @Override
+        public int calculate(int operand1, int operand2) {
+            return operand1 * operand2;
+        }
+    }, DIVISION("/"){
+        @Override
+        public int calculate(int operand1, int operand2) {
+            if(operand2 == 0)
+                throw new ArithmeticException("0으로 나눌 수 없습니다.");
+            return operand1 / operand2;
+        }
+    };
+
+    private String operator;
+    Operator(String operator){
+        this.operator = operator;
+    }
+    public abstract int calculate(int operand1, int operand2);
+}
```

Comment:
```suggestion
	public static Operator of(String operator){
		for(Operator op : Operator.values()){
			if(op.operator.equals(operator))
				return op;
		}
		throw new IllegalArgumentException("사칙연산 기호가 아닙니다.");
	}
}
```

열거 타입을 활용하려고 한다면 Operator 라는 열거 타입에 "연산자" 와 연산자에 매핑되는 "연산처리" 구현체를 호출하여 연산 결과를 반환해야 합니다.

연산자를 찾고, 특정 연산자에 대한 처리를 수행한다는 두 가지 작업을 처리 해야하는데 이미 `public abstract int calculate(int operand1, int operand2);`를 구현하셨기 때문에 연산자를 찾는 메서드만 만들어주시면 될 것 같습니다.

아래는St stream 으로 변환해본 내용입니다.

```suggestion
	public static Operator of(String operator){
		return Arrays.stream(values())
			.filter(o -> o.operator.equals(operator))
			.findAny()
			.orElseThrow(() -> new IllegalArgumentException("사칙연산 기호가 아닙니다."));
	}
}
```